src/alto-ui/Form/DatePicker/DatePicker.js

```diff
@@ -117,8 +139,9 @@ class DatePicker extends React.Component {
           {...this.props.inputProps}
           readOnly={this.props.readOnly}
           onFocus={this.handleFocus}
-          onChange={() => {}}
-          value={this.formatTextfieldDate()}
+          onChange={this.handleChangeInput}
+          onBlur={e => value && this.handleBlur(e)}
```

Comment:
why this condition here ? do it in method then.
+ do not forget to call `inputProps .obBlur` if it's a function

Reply:
When we define onBlur in our DatePicker, than DayPicker onDayClick would be called, because when we select day, onBlur fired first and re-render DayPicker component.
So, we should use our formating `handleBlur` only if we have a value in state.

inputProps.onBlur is called in `handleBlur` method

Reply:
I get it but just do `if(!value) return null;` in `handleBlur`. Same result then.

Reply:
Agree, fixed